pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -36,9 +36,9 @@ var (
 	// ErrVolumeZoneConflict is used for NoVolumeZoneConflict predicate error.
 	ErrVolumeZoneConflict = newPredicateFailureError("NoVolumeZoneConflict", "node(s) had no available volume zone")
 	// ErrNodeSelectorNotMatch is used for MatchNodeSelector predicate error.
-	ErrNodeSelectorNotMatch = newPredicateFailureError("MatchNodeSelector", "node(s) didn't match node selector")
```

Comment:
This changes the predicate name and not the description. 

`PredicateFailureError` struct looks as follows:
```go
type PredicateFailureError struct {
	PredicateName string
	PredicateDesc string
}
```
Also, the description seems clear enough:
`node(s) didn't match node selector`

Reply:
![image](https://user-images.githubusercontent.com/28048077/65980863-8fd22d00-e4aa-11e9-9438-fdfdc6f45e70.png)
this is the message I get when there is no fitable node to schedule. I can understand 「Insufficient memory」、「NodeUnschedulable」so the node cann't been scheduled. But 「MatchNodeSelector」so  the node cann't been scheduled? Shouldn't it because 「NodeSelectorNotMatch」so  the node cann't been scheduled?

Reply:
Can you show the beginning of this log message?

Reply:
![image](https://user-images.githubusercontent.com/28048077/66044669-adf46780-e554-11e9-920f-fcdad7fbde77.png)

Reply:
May I ask how is the following output produced? Is it a custom log that you added? Is `SCHED_FAILED_REASON` printing the `FailedPredicates` values?